lerna.json

```diff
@@ -1,11 +1,5 @@
 {
-    "packages": [
-        "linters/*",
-        "packages/*",
-        "projects/*",
-        "tools/*",
-        "react/*"
-    ],
+    "packages": ["linters/*", "packages/*", "projects/*", "tools/*", "react/*"],
```

Comment:
I don't know why some files were submitted together with my PR, they seem to be changes after linting...